app/models/item.rb

```diff
@@ -10,4 +10,14 @@ class Item < ApplicationRecord
   :url => "/images/:id/:style/:filename"
 	validates_attachment_content_type :image, content_type: ['image/jpeg', 'image/jpg', 'image/gif', 'image/png']
 	enum condition: ["active", "retired"]
+
+
+	def self.items_sold_by_title
+		group(:title).joins(:order_items).sum(:quantity)
+	end
```

Comment:
This method can probably be named better. Item and title pretty much mean the same thing. I think `total_sold_by item` might be a better name for this method.